src/app/tasks/modals/grade-task-modal/grade-task-modal.component.html

```diff
@@ -0,0 +1,68 @@
+<mat-card class="grade-task-modal">
+  <mat-card-title> Assess Task Quality </mat-card-title>
+  <mat-card-content>
+    <!-- For Grade Tasks -->
+    <div *ngIf="task.definition.is_graded">
+      <p>
+        Please provide a grade for task
+        <label class="task-label">
+          {{ task.definition.abbreviation }}
+        </label>
+      </p>
+      <div class="grade-toggle-group">
+        <mat-button-toggle-group #group="matButtonToggleGroup" (change)="updateGrade($event.value)">
+          <mat-button-toggle
+            *ngFor="let idx of gradeValues"
+            class="grade-toggle"
+            value="{{ idx }}"
+            aria-label="{{ grades[idx] }}"
+            matTooltip="This student completed the task to a {{ grades[idx] }} standard"
+            [matTooltipShowDelay]="500"
+            aria-describedby="This student completed the task to a {{ grades[idx] }} standard"
+          >
+            <grade-icon [grade]="grades[idx]"></grade-icon>
+          </mat-button-toggle>
+        </mat-button-toggle-group>
+      </div>
+    </div>
+    <!-- For Quality Points Tasks -->
+    <div
+      *ngIf="task.definition.max_quality_pts > 0"
+      [ngStyle]="{ 'margin-top': task.definition.is_graded && task.definition.max_quality_pts > 0 ? '1.75rem' : '0' }"
+    >
+      <p>
+        Please provide a quality rating for task
+        <label class="task-label">
+          {{ task.definition.abbreviation }}
+        </label>
+      </p>
+      <mat-slider
+        class="slider"
+        (input)="updateRating($event.value)"
+        [value]="rating"
+        [max]="totalRating"
+        [min]="1"
+        [step]="1"
+        [thumbLabel]="true"
+        [tickInterval]="auto"
+        aria-labelledby="task-rating-slider"
+      >
+      </mat-slider>
+      <div class="label-rating">
+        <p>Rating: {{ rating }} / {{ totalRating }}</p>
+      </div>
+    </div>
+  </mat-card-content>
+  <mat-divider></mat-divider>
+  <mat-card-actions class="buttons">
```

Comment:
**suggestion:** We can use the align attribute the drop the CSS class.

```suggestion
  <mat-card-actions align="end">
```

Reply:
I didn't know about this, thanks for sharing!